async-inference/Async-Inference-Walkthrough-SageMaker-Python-SDK.ipynb

```diff
@@ -0,0 +1,652 @@
+{
```

Comment:
<p>No need to explicitly attach an S3 policy, this is included by default in SageMakerFullAccess.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Done.  Kept (Optional) Amazon SNS access